src/visualization/utilities.jl

```diff
@@ -880,6 +880,41 @@ function unstructured_3d_to_1d_curve(nodes, data, curve, slice, point, nvisnodes
   return calc_arc_length(curve), data_on_curve, mesh_vertices_x
 end
 
+# Convert 3d unstructured data from a general mesh to 2d data at a plane.
+function unstructured_3d_to_2d_plane(nodes, data; elevations=nothing, slice=:yz, point=[0,0,0], nvisnodes=8)
```

Comment:
Why `nvisnodes=8`?

Reply:
Oh, this was just for testing. This will change it to 32 aswell.

Reply:
Could you also add a comment on this default value here, please?